spec/api/v1/teams_spec.rb

```diff
@@ -131,10 +131,26 @@
 
     subject { get "/api/v1/teams/#{team.slug}/users", params: { per_page: 2, api_key: api_token.token } }
 
-    it 'returns 2 team users' do
-      subject
+    context 'when team returns some users' do
+      it 'returns 2 team users' do
+        subject
+
+        expect(JSON.parse(response.body).count).to eq 2
+      end
+    end
+
+    context 'when team does not return any user' do
+      before do
+        team.users.destroy_all
 
-      expect(JSON.parse(response.body).count).to eq(2)
+        team.save
```

Comment:
I think you can remove this line

Reply:
You are right, `destroy_all` will already remove the relationship of the users object with the team